api/swagger.yaml

```diff
@@ -4802,6 +4802,50 @@ paths:
           type: "string"
           required: true
       tags: ["Image"]
+  /images/{name}/manifest:
+    get:
+      summary: "Get image manifest"
```

Comment:
@aaronlehmann yeah, I still need to update `swagger.yml`, but was holding off until we agreed on the endpoint and return values.